account_invoice_discount_date/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+{
+    "name": "Account Invoice Discount Date",
+    "summary": "Set the early discount date on invoices",
+    "version": "15.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Accounting",
+    "website": "https://github.com/OCA/account-invoicing",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "maintainers": ["Shide", "rafaelbn"],
+    "license": "AGPL-3",
```

Comment:
According to our internal contribution guidelines, this should be LGPL-3

Reply:
All modles have AGPL-3